docs/UserGuide.md

```diff
@@ -3,21 +3,43 @@ layout: page
 title: User Guide
 ---
 
-ModuleMateFinder Level 3 (MMF3) is a **desktop app for managing contacts in your educational organization, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, MMF3 can get your contact management tasks done faster than traditional GUI apps.
+ModuleMateFinder (MMF) is a **desktop app for managing contacts in your educational organization, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, MMF can get your contact management tasks done faster than traditional GUI apps.
 
 - [Quick Start](#quick-start)
 - [Features](#features)
-- [Quick Jump](#quick-jump)
 - [FAQ](#faq)
 - [Command Summary](#command-summary)
 
 --------------------------------------------------------------------------------------------------------------------
+## Quick Jump to Specific Features
```

Comment:
I think "Shortcut" might be better than "Quick Jump", especially since already have "Quick Start" nearby.

Reply:
Good point. Will change.